File: android-kiosco/app/src/main/java/com/hoteldelvalle/kiosco/Order.java
package com.hoteldelvalle.kiosco;

public class Order {
    private int id;
    private String roomNumber;
    private String product;
    private String roomType;
    private String guestName;
    private String checkIn;
    private String checkOut;
    private String status;
    private double amount;

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }
    public String getRoomNumber() { return roomNumber; }
    public void setRoomNumber(String roomNumber) { this.roomNumber = roomNumber; }
    public String getProduct() { return product; }
    public void setProduct(String product) { this.product = product; }
    public String getRoomType() { return roomType; }
    public void setRoomType(String roomType) { this.roomType = roomType; }
    public String getGuestName() { return guestName; }
    public void setGuestName(String guestName) { this.guestName = guestName; }
    public String getCheckIn() { return checkIn; }
    public void setCheckIn(String checkIn) { this.checkIn = checkIn; }
    public String getCheckOut() { return checkOut; }
    public void setCheckOut(String checkOut) { this.checkOut = checkOut; }
    public String getStatus() { return status; }
    public void setStatus(String status) { this.status = status; }
    public double getAmount() { return amount; }
    public void setAmount(double amount) { this.amount = amount; }
}

File: android-kiosco/app/src/main/java/com/hoteldelvalle/kiosco/data/model/Order.kt
// Dep: kotlinx.serialization
package com.hoteldelvalle.kiosco.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Order(
    val id: Int,
    @SerialName("room_number") val roomNumber: String? = null,
    val product: String? = null,
    @SerialName("room_type") val roomType: String? = null,
    @SerialName("guest_name") val guestName: String? = null,
    @SerialName("check_in") val checkIn: String? = null,
    @SerialName("check_out") val checkOut: String? = null,
    val status: String? = null,
    val amount: Double? = null
)

@Serializable
data class OrderRequest(
    val product: String,
    @SerialName("room_type") val roomType: String,
    @SerialName("guest_name") val guestName: String,
    @SerialName("id_document") val idDocument: String? = null,
    @SerialName("client_ref") val clientRef: String,
    val extra: String? = null,
    val days: Int? = null
)

@Serializable
data class OrderResponse(
    val order: Order? = null,
    val error: String? = null
)
